sdk/python/lib/test/test_next_serialize.py

```diff
@@ -104,7 +104,7 @@ def tearDown(self):
         self.assertEqual("hello.txt", prop["path"])
 
     @async_test
-    async def test_file_asset(self):
+    async def test_remote_asset(self):
```

Comment:
This fix is unrelated, this test was accidentally named the same as the above test. This works, for esoteric Python reasons, but it confused my code editor.